### DISCUSSION TITLE
npm versions no longer supported

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [babel/babel#16606] [Q&A] [answered]
Is it possible to somehow update dependencies that are no longer supported or there is no point in this?
On the other hand it’s not for nothing that this message is displayed

For example:
`npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported`
`npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.`
 etc

**Comment by JLHwung** [comment, marked as answer]
Babel 7 still supports Node.js 6, we can not bump glob to higher version in Babel 7 because glob@8 does not support Node.js 6. This issue will be fixed in Babel 8 which only supports recent node versions. You can try to upgrade to the Babel 8 alpha version `8.0.0-alpha.11`. this repo is using Babel 8 alpha as well.

**Reply by MyNameTony1000** [reply]
Hello, thanks for the quick response) I will wait for version 8
